README.md

```diff
@@ -13,9 +13,25 @@ Installation & usage
     $ composer require --dev sylius-labs/coding-standard
     ```
     
-2. Include a configuration file in your `easy-coding-standard.yml`:
+2. Import the configuration file in your `ecs.php`:
```

Comment:
`ecs.php` -> `easy-coding-standard.php`? no need to change the name

Reply:
Correct, but as we are changing the extension already, I figured we can follow the official ECS docs for the filename: https://github.com/symplify/easy-coding-standard#usage